app/components/mandaat/burgemeester-selector.js

```diff
@@ -79,26 +110,47 @@ export default class MandaatBurgemeesterSelectorComponent extends Component {
     return newMandataris;
   }
 
-  async loadBurgemeesterPersoon(burgemeesterMandaat) {
-    const burgemeesters = await burgemeesterMandaat.bekleedDoor;
+  async loadBurgemeesterPersoon() {
+    const burgemeesters = await this.burgemeesterMandate.bekleedDoor;
+    const voorzitterVastBureau =
+      await this.voorzitterVastBureauMandate.bekleedDoor;
+
+    const targetMandatarisses = [];
 
     if (burgemeesters.length === 0) {
-      this.mandataris = await this.createMandataris(burgemeesterMandaat);
-      return null;
+      targetMandatarisses.push(
+        await this.createMandataris(this.burgemeesterMandate)
+      );
     } else if (burgemeesters.length === 1) {
-      const burgemeester = burgemeesters[0];
-      this.mandataris = burgemeester;
-      return await burgemeester.isBestuurlijkeAliasVan;
+      targetMandatarisses.push(burgemeesters[0]);
     } else {
       await this.setMultipleBurgemeestersError(burgemeesters);
-      return burgemeesters[0].isBestuurlijkeAliasVan;
+      targetMandatarisses.push(burgemeesters[0]);
     }
+    if (voorzitterVastBureau.length === 0) {
+      targetMandatarisses.push(
+        await this.createMandataris(this.voorzitterVastBureauMandate)
+      );
+    } else if (voorzitterVastBureau.length === 1) {
+      targetMandatarisses.push(voorzitterVastBureau[0]);
+    } else {
+      await this.setMultipleBurgemeestersError(voorzitterVastBureau);
+      targetMandatarisses.push(voorzitterVastBureau[0]);
```

Comment:
I think this is targetMandatarisses is not super user friendly. It can be that you update the voorzitter without any warning. Or it can be that you have two voorzitters one that is the same as the burgemeester and another one as well. But if the burgemeester is the second result of the query, it will update the first voorzitter and not the one that is the same as the burgemeester.